ipet/validation/Validation.py

```diff
@@ -69,6 +69,29 @@ def __init__(self, solufilename : str = None, tol : float = DEFAULT_RELTOL, feas
 
         self.feastol = feastol
 
+    def set_tol(self, tol):
+        """sets this validation's tol attribute
+
+        Parameters
+        ----------
+
+        tol : float
+            new value for the tol for this validation
+        """
+        self.tol = tol
+
+    def set_feastol(self, feastol):
```

Comment:
Can you specify that both functions require a float attribute not only in the doc string?